controllers/user.controller.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const util = require('util');
+const mysql = require('mysql');
+const { response } = require('express');
+const { addUser } = require('../service/user.service');
+const sendOTP = require('../service/otp/sendOTP');
+const verifyOTP = require('../service/otp/verifyOTP');
+
+module.exports = {
+  addUser: async (req, res) => {
+    try {
+      var data = req.body;
+      const result = await addUser(data);
```

Comment:
We should check if the user existed then we shouldn't create user, it will cause duplicated users data (we should have uniq phone number constraint in the user table too). If there is a user existed, return this user data rather than create a new one.